### PR TITLE
Ming/brain cc

### PR DESCRIPTION
**Pull request by paulosaurusrex**

### CODE REVIEW

File: coordination/model/real/brain.py [hunk 1]
```diff
@@ -254,18 +256,34 @@ def _create_coordination(self, bundle):
             #  Also include parameters for us to able to set the values of individualism,
             #  coordination and common cause independently when use the newly implemented 3d
             #  coordination modules.
-            coordination = SigmoidGaussianCoordination(
```
Comment (paulosaurusrex) [inline coordination/model/real/brain.py]: We don't want to remove this completely. We still want to use SigmoidGaussianCoordination when common cause is False. You should add an if/else here for `bundle.common_cause`.

File: coordination/model/real/brain.py [hunk 2]
```diff
@@ -254,18 +256,34 @@ def _create_coordination(self, bundle):
...
+                    pymc_model=self.pymc_model,
+                    num_time_steps=bundle.num_time_steps_to_fit,
+                    mean_mean_uc0=bundle.mean_mean_uc0,
+                    sd_mean_uc0=bundle.sd_mean_uc0,
+                    sd_sd_uc=bundle.sd_sd_uc,
+                    mean_uc0=bundle.mean_uc0,
+                    sd_uc=bundle.sd_uc,
+                    initial_samples=bundle.initial_coordination_samples,
+                    unbounded_coordination_observed_values=given_coordination,
+                    include_common_cause=bundle.enable_common_cause
```
Comment (paulosaurusrex) [inline coordination/model/real/brain.py]: This parameter does not exist in the` DirichletGaussianCoordination3D` or `SigmoidGaussianCoordination3D`. please look at the constructors of these classes because the other parameters also changed names.

File: coordination/model/real/brain.py [hunk 3]
```diff
@@ -0,0 +1,786 @@
...
+            #  we use the sigmoid + normalization version.
+            if bundle.common_cause:
+                if bundle.coordination_mode == "dirichlet":
+                    coordination = DirichletGaussianCoordination3D(
+                        pymc_model=self.pymc_model,
+                        num_time_steps=bundle.num_time_steps_to_fit,
+                        mean_mean_uc0=bundle.mean_mean_uc0,
+                        sd_mean_uc0=bundle.sd_mean_uc0,
+                        initial_samples=bundle.initial_coordination_samples,
+                        unbounded_coordination_observed_values=bundle.observed_coordination_for_inference,
```
Comment (paulosaurusrex) [inline coordination/model/real/brain.py]: This should be given_coordination. You need to do the same thing for individualism and common_cause. Create given_individualism and given_common_cause as logits of the unbounded values passed to the config bundle.

File: coordination/model/real/brain.py [hunk 4]
```diff
@@ -0,0 +1,786 @@
...
+                        sd_uc_common_cause=bundle.sd_uc_common_cause,
+                    )
+                elif bundle.coordination_mode == "sigmoid":
+                    coordination = SigmoidGaussianCoordination3D(
+                        pymc_model=self.pymc_model,
+                        num_time_steps=bundle.num_time_steps_to_fit,
+                        mean_mean_uc0=bundle.mean_mean_uc0,
+                        sd_mean_uc0=bundle.sd_mean_uc0,
+                        initial_samples=bundle.initial_coordination_samples,
+                        unbounded_coordination_observed_values=bundle.observed_coordination_for_inference,
```
Comment (paulosaurusrex) [inline coordination/model/real/brain.py]: This should be given_coordination. You need to do the same thing for individualism and common_cause. Create given_individualism and given_common_cause as logits of the unbounded values passed to the config bundle.
